### PR TITLE
[camera-capture] Added support for multiple cameras

### PR DESCRIPTION
**Pull request by lostsource**
This commit includes changes requested in comments for a previous (separate) pull request https://github.com/GoogleChrome/chrome-app-samples/pull/377

### CODE REVIEW

File: samples/camera-capture/app.js [hunk 1]
```diff
@@ -23,10 +23,43 @@ Author: Eric Bidelman (ericbidelman@chromium.org)
  *
  * @see http://developer.chrome.com/apps/manifest.html#permissions
  */
+
```
Comment (beaufortfrancois) [inline samples/camera-capture/app.js]: Do you mind updating comment below mentioning we request both video and audio since this is not the case anymore?

Comment (beaufortfrancois) [inline samples/camera-capture/app.js]: We'll also need to move "audioCapture" permission from `manifest.json` file.

Comment (beaufortfrancois) [inline samples/camera-capture/app.js]: And update `README.md` file as well

**Comment by beaufortfrancois** [comment]
Can you also bump app version in the `manifest.json` file? 
When it's done, I'll merge your pull request.

**Comment by lostsource** [comment]
Removed audioCapture permission and other references to audio. Version bumped.

**Comment by beaufortfrancois** [comment]
nirringrazzjak ħafna!
Updated sample is now available at https://chrome.google.com/webstore/detail/camera-capture-sample/ilefapmpngkdnnllcnlcjffipbolhklf

**Comment by lostsource** [comment]
That must be the first time anyone wrote in Maltese on Github, merci!

**Pull request merged**